app/overrides/spree/admin/shared/_product_sub_menu/add_variant_overrides_tab.html.haml.deface

```diff
@@ -1,3 +1,3 @@
 / insert_bottom "[data-hook='admin_product_sub_tabs']"
 
-= tab :variant_overrides, label: "Inventory", url: main_app.admin_inventory_path, match_path: '/inventory'
```

Comment:
Adding the variant_overrides key fix this issue, I think there is an issue with the tab method regarding the translation.